Python/optimizer.c

```diff
@@ -569,7 +565,7 @@ translate_bytecode_to_trace(
             {
                 if (instr + 2 - oparg == initial_instr && code == initial_code) {
                     RESERVE(1, 0);
-                    ADD_TO_TRACE(_JUMP_TO_TOP, 0, 0);
+                    ADD_TO_TRACE(_JUMP_TO_TOP, 0, 0, 0);
```

Comment:
target is 0 because `_JUMP_TO_TOP` never deopts?

Reply:
Yes.